src/sentry/web/frontend/setup_wizard.py

```diff
@@ -117,7 +117,7 @@ def get_token(orgs: list[Organization], user: User):
             return token
 
     # Otherwise, generate a user token
-    tokens = ApiToken.objects.filter(user=user)
+    tokens = ApiToken.objects.filter(user_id=user.id)
```

Comment:
Having to use `user_id` instead of `user` as a filter when you're working with the request.user is something we should tell folks more broadly about.

Reply:
Yup, that will be part of this rollout for sure.

Reply:
Notably, after this PR lands, tests will reflect RpcUser and these issues will become more obvious immediately, even if the rollout in production is staged.